File: TradeItIosEmsApi/TradeItOAuthDeleteLinkRequest.m
#import "TradeItOAuthDeleteLinkRequest.h"

@implementation TradeItOAuthDeleteLinkRequest

- (id)initWithApiKey:(NSString *)apiKey
       userId:(NSString *)userId
       userToken:(NSString *)userToken {
    self = [super init];
    
    if (self) {
        self.apiKey = apiKey;
        self.userId = userId;
        self.userToken = userToken;
    }
    
    return self;
}

@end

File: TradeItIosEmsApi/TradeItOAuthDeleteLinkRequest.swift
class TradeItOAuthDeleteLinkRequest: TradeItRequest {
    var apiKey: String
    var userId: String
    var userToken: String?
    
    init(apiKey: String, userId: String, userToken: String?) {
        self.apiKey = apiKey
        self.userId = userId
        self.userToken = userToken
    }
}
